pkg/engine/background.go

```diff
@@ -69,12 +70,30 @@ func filterRule(rclient registryclient.Client, rule kyvernov1.Rule, policyContex
 		return nil
 	}
 
+	// check if there is a corresponding policy exception
+	exception, err := matchesException(policyContext, &rule)
+	// if we found an exception
+	if err == nil && exception != nil {
+		key, err := cache.MetaNamespaceKeyFunc(exception)
+		logger := logging.WithName("exception")
+		// TODO: increase metrics
```

Comment:
what does those comment mean? do we need to increment the rule metrics or something else?

Reply:
This is recommended by @eddycharly . 
I assume this is to instrument the triggered exception count.

cc: @eddycharly  Pls fix me if I'm understanding wrong, or anything you'd like to point out.